packages/graph-explorer/src/modules/GraphViewer/renderNode.tsx

```diff
@@ -52,9 +73,7 @@ export async function renderNode(
  */
 function embedSvgInsideCytoscapeSvgWrapper(svgContent: string): string {
   return `<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg>
-<svg width="24" height="24" xmlns="http://www.w3.org/2000/svg">
-  ${svgContent}
-</svg>`;
+${svgContent}`;
```

Comment:
I manually tested the change to remove the extraneous wrapper `svg` element. I'm not sure why I thought it was needed in the past. But not having it seems to work with all shapes and sizes of SVG images. So thank you for catching that.

However, the tests do need to be updated to account for this change.